<launch>
  <rosparam file="$(find pr2_offset_calib_controller)/config/pr2_offset_calib_controller.yaml" command="load" />
  <node pkg="pr2_controller_manager" type="spawner" args="pr2_offset_calib_controller/OffsetCalibrationController" name="pr2_offset_calib_controller_spawner" />
</launch>
